framework/auth/cas.py

```diff
@@ -208,15 +213,23 @@ def get_logout_url(*args, **kwargs):
     """
     return get_client().get_logout_url(*args, **kwargs)
 
+
 def get_profile_url():
     """Convenience function for getting a profile URL for a user.
     """
     return get_client().get_profile_url()
 
+
 def make_response_from_ticket(ticket, service_url):
-    """Given a CAS ticket and service URL, attempt to the user and return a proper
-    redirect response.
     """
+    Given a CAS ticket and service URL, attempt to the user and return a proper redirect response.
+
+    :param ticket:
+    :param service_url:
+    :param user_agent:
```

Comment:
You are right. Working on a clean-up right now. I used to have it for IE9 compatibility.